backend/inspirehep/records/marshmallow/literature/common/author.py

```diff
@@ -56,20 +57,23 @@ class AuthorSchemaV1(FirstAuthorSchemaV1):
     record = fields.Raw()
     signature_block = fields.Raw()
     uuid = fields.Raw()
-    bai = fields.Method("get_bai", dump_only=True)
+    ids = fields.Method("get_ids", dump_only=True)
 
     @staticmethod
-    def get_bai(data):
+    def get_ids(data):
```

Comment:
since we are using ` data.get("ids", [])` in multiple places let's access it once

Reply:
yes, my idea was do drop the whole code block after finishing all the operations, but I can move it to the variable